.dockerignore

```diff
@@ -10,4 +10,3 @@ site-packages
 venv
 webpack-stats.json
 webpack-stats-actions.json
-Dockerfile
```

Comment:
Turns out [you can't actually ignore `Dockerfile`](https://www.ctl.io/developers/blog/post/more-docker-image-cache-tips/), since it is needed for the build. Including it here does nothing.